config/spotbugs-exclude.xml

```diff
@@ -273,4 +273,11 @@
     </Or>
     <Bug pattern="SPP_PASSING_THIS_AS_PARM"/>
   </Match>
+  <Match>
+    <!-- false positive, we are storing file contents in a field
```

Comment:
Won't fix.

Issue is detected properly, problem is that our design requires operating over string field of class

Reply:
@stoyanK7 after a second look, I do not understand why we don't use a local variable for `sourceContent`. Why do we need a field at all?